### PR TITLE
CORTX-32833: resolve codacy issues from provisioner repo

### PR DESCRIPTION
**Pull request by tanujashinde0405**
Signed-off-by: Tanuja Shinde <tanuja.shinde@seagate.com>

# Problem Statement

*   resolve codacy issues from provisioner repo

## Design

*   For Bug, Describe the fix here.
*   For Feature, Post the link for design

## Coding

Checklist for Author

*   \[x] Coding conventions are followed and code is consistent

## Testing

Checklist for Author

*   \[x] Unit and System Tests are added
*   \[ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
*   \[x] Testing was performed with RPM

## Impact Analysis

Checklist for Author/Reviewer/GateKeeper

*   \[ ] Interface change (if any) are documented
*   \[ ] Side effects on other features (deployment/upgrade)
*   \[ ] Dependencies on other component(s)

## Review Checklist

Checklist for Author

*   \[x] JIRA number/GitHub Issue added to PR
*   \[x] PR is self reviewed
*   \[x] Jira and state/status is updated and JIRA is updated with PR link
*   \[x] Check if the description is clear and explained

## Documentation

Checklist for Author

*   \[ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: test/setup/test_multiple_storage_sets.py [hunk 1]
```diff
@@ -77,5 +77,6 @@ def test_for_multiple_storage_sets(self):
         self.assertEqual(rc, 0)
         os.remove(cluster_conf_file_path)
 
+
```
Comment (nitin-seagate) [inline test/setup/test_multiple_storage_sets.py:80]: Add new line to file

**Review by nitin-seagate — APPROVED** [review]
LGTM

File: test/setup/test_multiple_storage_sets.py (hunk 1 above)
Comment (tanujashinde0405) [reply test/setup/test_multiple_storage_sets.py:80]: done

**Review by jaikumargidwani — APPROVED** [review]
LGTM

**Pull request merged**